config/sync/block.block.footerplayland.yml

```diff
@@ -0,0 +1,25 @@
+uuid: 268f638a-1d0c-41c8-829f-310bc4c53c80
+langcode: en
+status: true
+dependencies:
+  content:
+    - 'block_content:basic:b8758650-346d-4a08-b5e3-082c7bbeb14c'
```

Comment:
This block won't work in other environments because it's connected to a content block within you localhost. You should remove this config file (only the config file, not the block) and add its name to the `config_ignore` module's settings, so that it won't be part of your site config and you can create this block manually in each environment.

Reply:
and don't forget to remove this file from git :)